### DISCUSSION TITLE
Welcome to anuraggarg3.github.io Discussions!

### DISCUSSION THREAD

**Discussion by anuraggarg3** [anuraggarg3/anuraggarg3.github.io#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Tanmay-Joshi73** [comment]
HHey there, there is a documentation issue on your website. If you don't mind, I can solve it for you
